### ISSUE TITLE
Undefined variable: configuration inside Model_crud.php

### ISSUE THREAD

**Issue by jadolyo** [ridwanskaterock/cicool#425]
A PHP Error was encountered
Severity: Notice

Message: Undefined variable: configuration

Filename: models/Model_crud.php

Line Number: 140

Backtrace:

File: D:\xampp72\htdocs\erc\modules\crud\models\Model_crud.php
Line: 140
Function: _error_handler

File: D:\xampp72\htdocs\erc\modules\crud\controllers\backend\Crud.php
Line: 125
Function: get_crud_field_configuration

File: D:\xampp72\htdocs\erc\index.php
Line: 339
Function: require_once

![image](https://user-images.githubusercontent.com/5902217/95656100-3f090180-0b0c-11eb-8d81-5c18c48d588b.png)

**Comment by ridwanskaterock** [comment]
what version are you using ?

can you screenshoot file /application/models/Model_crud.php line 140 

![image](https://user-images.githubusercontent.com/9270445/95722393-11ec5880-0c9e-11eb-9589-3b639ffac803.png)

**Comment by jadolyo** [comment]
![image](https://user-images.githubusercontent.com/5902217/95771716-cea0e280-0cbb-11eb-8d7c-5bcf802fbb12.png)

**Comment by jadolyo** [comment]
Yes that was weird I didn't notice that $validation should be $configuraion

There is another problem too in select

![image](https://user-images.githubusercontent.com/5902217/95772223-ac5b9480-0cbc-11eb-945c-8e54418561fe.png)
